packages/open-truss/src/components/OTUqiDataProvider.tsx

```diff
@@ -18,15 +18,14 @@ export const Props = BaseOpenTrussComponentV1PropsShape.extend({
   ...withChildren,
   source: StringSignal,
   query: StringSignal,
-  force_query: NumberSignal,
+  forceQuery: NumberSignal,
```

Comment:
We'll need to remember to update our configs in our internal app when we pull in this change.

Reply:
Typescript will tell you 😅